rustfmt-core/rustfmt-lib/src/formatting/comment.rs

```diff
@@ -1580,17 +1582,6 @@ pub(crate) fn recover_comment_removed(
     let snippet = context.snippet(span);
     let includes_comment = contains_comment(snippet);
     if snippet != new && includes_comment && changed_comment_content(snippet, &new) {
-        // We missed some comments. Warn and keep the original text.
-        if context.config.error_on_unformatted() {
```

Comment:
Are we no longer going to warn in this scenario? (I may just not recall or may not have seen other changes that handle this scenario elsewhere)